cellmlmanip/model.py

```diff
@@ -567,6 +568,65 @@ def get_symbol_by_cmeta_id(self, cmeta_id):
 
         raise KeyError('No variable with cmeta id "%s" found.' % str(cmeta_id))
 
+    def get_symbol_by_ontology_term(self, namespace, local_name):
+        """
+        Searches the RDF graph for a variable annotated with the given
+        ``namespace:local_name`` and returns its symbol.
+
+        Specifically, this method searches for a unique variable annotated with
+        predicate ``http://biomodels.net/biology-qualifiers/is`` and the object
+        specified by ``namespace + local_name``.
+
+        Will raise a ``KeyError`` if no variable with the given annotation is
+        found, and a ``RuntimeError`` if more than one variable with the given
```

Comment:
`RuntimeError` is a bit generic. Would `ValueError` be better?

Reply:
I suppose... If it's legal to have multiple things annotated with "is", but not to request one and insist it's unique...

Reply:
It's slightly tenuous, but I don't think would surprise users!